### DISCUSSION TITLE
Custom Image Transition in Next.js With Tailwind CSS — Jagad Yudha Awali

### DISCUSSION THREAD

**Discussion by giscus** [jagadyudha/jagad.dev#50] [Announcements]
# Custom Image Transition in Next.js With Tailwind CSS — Jagad Yudha Awali

make a custom more interactive with the use of CSS transitions.

https://jagad.dev/posts/custom-image-transition-in-nextjs-with-tailwind-css

**Comment by jagadyudha** [comment]
Discussion and feedback are appreciated 😁

**Comment by thinkjrs** [comment]
:fire: :fire: Absolutely love this, thank you @jagadyudha!

**Reply by jagadyudha** [reply]
Thanks 😀👍
